files/tubular/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "tubular",
+  "repo": "unosquare/tubular",
+  "files": {
+    "include": ["tubular-bundle.js", "tubular-bundle.min.js", "tubular-bundle.css","tubular-bundle.min.css"]
```

Comment:
You need to add a basePath parameter pointing to "dist/" https://github.com/unosquare/tubular/tree/master/dist

Also you dont have any tagged versions, our bot wont work until you start tagging them